src/views/states/list/hooks/useStateFilters.ts

```diff
@@ -30,14 +35,41 @@ export const useStateSearchFilters = (): RowSearchFilter<V1beta1NodeNetworkState
         const searchMACAddress = searchText?.selected?.[0];
         if (!searchMACAddress) return true;
 
-        const interfaces = obj?.status?.currentState
-          ?.interfaces as NodeNetworkConfigurationInterface[];
+        const interfaces = getInterfaces(obj);
 
         return interfaces?.some((iface) => searchInterfaceByMAC(searchMACAddress, iface));
       },
       filterGroupName: t('MAC address'),
       placeholder: t('Search by MAC address...'),
     },
+    {
+      type: FILTER_TYPES.LLDP_NAME,
+      filter: (searchText, obj) => {
+        const searchLLDPName = searchText?.selected?.[0];
+        if (isEmpty(searchLLDPName)) return true;
+
+        const interfaces = getInterfaces(obj);
```

Comment:
Nice to define and use `getInterfaces` here and in other places to simplify the code a bit :) 👍🏽